### PR TITLE
Add description for pd dashboard

### PR DESCRIPTION
**Pull request by YangKeao**
Add description for grafana pd dashboard.

This json file was generated by grafana, and some fields like "scopedVars" was removed.

### CODE REVIEW

**Comment by zhouqiang-cl** [comment]
metric addr is 172.16.5.175:3000

File: scripts/pd.json [hunk 1]
```diff
@@ -58,8 +58,10 @@
   "gnetId": null,
   "graphTooltip": 1,
   "id": null,
-  "iteration": 1564738811479,
-  "links": [],
+  "iteration": 1566379062374,
+  "links": [
```
Comment (rleungx) [inline scripts/pd.json]: Why there is a lot of changes like this?

Comment (YangKeao) [reply scripts/pd.json]: Sorry that's my fault. I will use `jq` to reformat this json file.

Comment (YangKeao) [reply scripts/pd.json]: Fixed

File: scripts/pd.json [hunk 2]
```diff
@@ -6508,6 +6916,7 @@
           "dashLength": 10,
           "dashes": false,
           "datasource": "${DS_TEST-CLUSTER}",
+          "description": "The count of the corresponding schedule commands which PD sends to each TiKV instance",
```
Comment (rleungx) [inline scripts/pd.json]: This description is wrong.

File: scripts/pd.json [hunk 3]
```diff
@@ -5544,6 +5897,7 @@
           "dashLength": 10,
           "dashes": false,
           "datasource": "${DS_TEST-CLUSTER}",
+          "description": "The count of etcd transactions",
```
Comment (rleungx) [inline scripts/pd.json]: Should it be the rate?

File: scripts/pd.json (hunk 2 above)
Comment (YangKeao) [reply scripts/pd.json]: It has been changed to "The heartbeat latency of each TiKV instance in .99"

File: scripts/pd.json (hunk 3 above)
Comment (YangKeao) [reply scripts/pd.json]: The title of it is "Handle transactions count". But the target is `sum(rate(pd_txn_handle_txns_duration_seconds_count[5m])) by (instance, result)`

I will change both description and title into "rate".

**Comment by YangKeao** [comment]
@rleungx friendly ping. Do you have more suggestion on this PR?

**Comment by YangKeao** [comment]
Now only description related change was left.

**Comment by rleungx** [comment]
Does this PR include all metrics? @YangKeao

**Comment by YangKeao** [comment]
@rleungx It includes all metrics mentioned in [documents](https://github.com/pingcap/docs/tree/master/dev/reference/key-monitoring-metrics). 

But as I know more changes on this documents will be landed these days. So description may also be changed these days.

**Comment by rleungx** [comment]
Do we need to merge this PR first or just wait for the change landed?

**Comment by YangKeao** [comment]
@rleungx That's OK, as I found that other PRs (for tikv dashboard and overview dashboard) have been merged.

**Pull request merged**